lib/unleash/context.rb

```diff
@@ -19,8 +19,11 @@ def initialize(params = {})
     end
 
     def to_s
-      "<Context: user_id=#{@user_id},session_id=#{@session_id},remote_address=#{@remote_address},properties=#{@properties}" \
-        ",app_name=#{@app_name},environment=#{@environment}>"
+      formatted_attributes = instance_variables.map do |var|
+        "#{var.to_s.delete('@')}=#{instance_variable_get(var)}"
+      end
+
+      "<Context: #{formatted_attributes.join(', ')}>"
```

Comment:
If one were to use meta-programming, i'd have preferred if the built in `to_h` method was used instead:

```suggestion
      "<Context: " + self.to_h.map{|k,v| "#{k}=#{v}"}.join(', ') + ">"
```

Reply:
I had no idea to use the `to_h` method. It is a very nice and simple implementation!